.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+<!-- IMPORTANT
+Before creating an issue, please check that your version of Tarchive is up to date and that there is no other open issue.
+If you are making a feature request please include a description of said feature as well as examples if necessary.
+If you are reporting a bug please include the following:
+-->
+- **Intended behaviour**
+
+- **Actual behaviour**
+
+- **How to reproduce**
```

Comment:
I think it's not clear enough wether or not the issue submitter is supposed to write something inbetween the bulletpoints.

So is this just a checklist? Or is the issuer supposed to write something inbetween the bulletpoints? If yes, then I'd add another newline inbetween them or maybe even have an example text like this:
```
- **Intended behaviour**
<Describe the intended behaviour here.>
```